pkg/core/server/server.go

```diff
@@ -55,9 +64,22 @@ func (s *Server) Handle(path string, hf handler.Factory) *mux.Route {
 
 		h := hf.NewHandler(r.Context(), configuration)
 
-		h.Handle(handler.Context{
-			ResponseWriter: rw,
-			Request:        r,
-		})
+		var authInfo auth.AuthInfo
+		if s.authInfoResolverFactory != nil {
+			resolver := s.authInfoResolverFactory.NewResolver(r.Context(), configuration)
+			authInfo, _ = resolver.Resolve(r)
+		}
```

Comment:
Assume `s.authInfoResolverFactory` is required.

Make the default factory in `NewServer`.

Reply:
But the default requires `authDependency` to work, which the server itself has no knowledge of it.